README.md

```diff
@@ -444,16 +444,15 @@ To use the JAR with no dependencies in a Java application, the following require
 Javadoc JAR builds with `mvn install` alongside the other JAR files. To extract the Javadoc HTML files, use the following command: `jar -xvf amazon-timestream-jdbc-1.0.0-javadoc.jar`
 
 ### Known Issues
-1. Timestream does not support fully qualified table names. Tools like Tableau may not work as expected.
+1. Timestream does not support fully qualified table names.
```

Comment:
This statement could be little confusing for customers, especially when reading one of the caveats `Timestream JDBC driver only works with queries with fully qualified table names.`

Reply:
Yes it is confusing. After this change we could give fully qualified table names for Tableau. `Timestream JDBC driver only works with queries with fully qualified table names.` is removed.